spec/requests/care_plan_contacts_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+RSpec.describe 'CarePlanContacts', type: :request do
+  let(:user) { create(:user) }
+
+  describe '#create' do
+    context 'when the user is logged in' do
+      before { sign_in user }
+
+      context 'when valid params are supplied' do
+        it 'creates the care plan contact' do
+          valid_params = { name: 'Person Name', phone: '416000000' }
+
+          post care_plan_contacts_create_path, params: { care_plan_contact: valid_params }
```

Comment:
is `valid_params` used anywhere is else? if not, I'd say you can refactor into one line.

Don't really _have_ to make everything into it's own variable
```suggestion
          post care_plan_contacts_create_path, params: { 
            care_plan_contact:  { 
              name: 'Person Name', phone: '416000000' 
            } 
          }
```